2_process/src/create_group_task_plan.R

```diff
@@ -48,9 +51,9 @@ create_group_tasks <- function(task_ids, log_folder, irradiance_zips, clarity_zi
     } 
   )
   combine_pb0_zipped_files <- scipiper::create_task_step(
```

Comment:
Probably not worth updating the object name here now that the whole pipeline has been run. Although, change that shouldn't actually result in a rebuild.

Reply:
it actually won't trigger a rebuild of all the files since the functions and inputs remain the same. This step name is not used other than the internals of creating a task plan. I will make the change and double check though.